### ISSUE TITLE
Request timeout, automation failing

### ISSUE THREAD

**Issue by thkrmr** [Hyundai-Kia-Connect/kia_uvo#862]
**Region and Brand of car**
EU KIA Xceed PHEV '23

**Version of the integration**
v2.24.2

**Describe the bug**
Automations and scripts are failing hard when the force update (refresh) service call times out, continue on error does not work.
Automation execution is interrupted when this error occurs. Fails regardless of a target vehicle being specified or not. 
Timeout doesn't occur when car has cellular reception. 

**Debug logs if an error occurred**
[debug.log](https://github.com/Hyundai-Kia-Connect/kia_uvo/files/14939148/debug.log)

**To Reproduce**
Park car in underground garage with no cellular reception. Call force update service.

**Expected behavior**
Fail gracefully, so that either continue on error works OR the service call timeout doesn't kill the automation execution.

**Screenshots**
![Screenshot 2024-04-11 at 02 11 44](https://github.com/Hyundai-Kia-Connect/kia_uvo/assets/7042350/c8f62159-e693-4615-a180-b93b68af8963)

**Additional context**
"Regular" update is cached info and doesn't give the actual status.
Force update is necessary to know whether the car has been plugged this moment.

**Comment by opb** [comment]
I have an automation which has just started failing over te last day or so. The message I get from homeassistant is:

`The automation "Kia not plugged in" (automation.kia_not_plugged_in) has an action that calls an unknown service: kia_uvo.force_update.`

Perhaps in the underlying library has changed

**Comment by thkrmr** [comment]
> I have an automation which has just started failing over te last day or so. The message I get from homeassistant is:
> 
> `The automation "Kia not plugged in" (automation.kia_not_plugged_in) has an action that calls an unknown service: kia_uvo.force_update.`
> 
> Perhaps in the underlying library has changed

I can call the service just fine, force update is there, it times out as seen in my logs. Timeout exception isn't being caught and thus cascades to an automation also throwing an exception and stopping.

**Comment by cdnninja** [comment]
It's shouldn't be caught.  As the call failed. If we caught and ignored it you wouldn't know it didn't work.

**Comment by fuatakgun** [comment]
I agree with @cdnninja that we should raise this issue to the user so owner can learn about it.
I am more interested why automation/script is not moving on even though you are using `continue on error`.
I will deep dive on this.

**Comment by thkrmr** [comment]
@fuatakgun the issue should definitely be raised, as I wrote though, failing gracefully should be made possible, continuing on error if desired.

**Comment by cdnninja** [comment]
To confirm in your automation YAML you have continue on error enabled? 

This issue is more of a home assistant issue than kia_uvo.

**Comment by MivraMe** [comment]
I don't know if it can help
![Screenshot_20240422-162207.png](https://github.com/Hyundai-Kia-Connect/kia_uvo/assets/88721578/42f69573-ee69-45a6-aafb-0ac3c78539c3)

**Comment by thkrmr** [comment]
@cdnninja Continue on error is enabled.

**Comment by cdnninja** [comment]
Could you post your YAML?
